src/tests/databases/core_8/meta.txt

```diff
@@ -19,4 +19,5 @@
 19	1	genebuild.sample_location	KB871578.1:9766653-9817473
 20	1	strain.type	test
 23	1	genebuild.provider_name	test
-24	1	genebuild.start_date	test
\ No newline at end of file
+24	1	genebuild.start_date	test
```

Comment:
Start_date in test.... should be a date. at least.

Reply:
Replaced with existing data